src/pages/Branches.tsx

```
@@ -4,20 +4,50 @@ import { useQuery, useMutation } from '@tanstack/react-query'
+            actions={[
+              <Button
+                type="link"
+                onClick={async () => {
+                  const r = await idx.mutateAsync(b.name)
+                  message.success(
+                    `索引完成：routes=${r.indexed}, owners=${r.owners}, files=${r.files}`
+                  )
+                }}
+              >
+                API 索引
+              </Button>,
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)